Monika After Story/game/script-topics.rpy

```diff
@@ -12905,3 +12905,28 @@ label monika_enjoyingspring:
     else:
         m 2rkc "...but I guess there's no real way to avoid it, is there?"
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_unknown",category=['psychology'],prompt="Fear of the unknown",random=True))
+
+label monika_unknown:
+    m 3eud "[player], did you know that a lot of people are afraid of the dark?"
+    m 3eua "Even though it's often dismissed as a childish fear, it's not that uncommon for adults to suffer from it as well."
+    m 4eub "Fear of the dark, called 'nyctophobia', is usually caused by the brain's exaggerated guess of what may be hiding in the shadows, rather than darkness itself."
+    m 4eua "We're scared because we don't know what's there...{w=1}even if it's usually nothing."
```

Comment:
I think fear of the dark is a good example, but if you ask me, I think it's important to keep in mind that it's not just about monsters popping up and getting you. 

Darkness reduces visibility, so it's also an inherent safety hazard if you bump into or trip over something because you can't see.

Reply:
Hadn't considered that, good thinking. (Working on it)